src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernel.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Kernel;
+
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Routing\RouteCollectionBuilder;
+
+/**
+ * A Kernel that provides configuration hooks.
+ *
+ * @author Ryan Weaver <ryan@knpuniversity.com>
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+abstract class MicroKernel extends Kernel
+{
+    /**
+     * Add or import routes into your application.
+     *
+     *     $routes->import('config/routing.yml');
+     *     $routes->add('/admin', 'AppBundle:Admin:dashboard', 'admin_dashboard');
+     *
+     * @param RouteCollectionBuilder $routes
+     */
+    abstract protected function configureRoutes(RouteCollectionBuilder $routes);
+
+    /**
+     * Configures the container.
+     *
+     * You can register extensions:
+     *
+     * $c->loadFromExtension('framework', array(
+     *     'secret' => '%secret%'
+     * ));
+     *
+     * Or services:
+     *
+     * $c->register('halloween', 'FooBundle\HalloweenProvider');
+     *
+     * Or parameters:
+     *
+     * $c->setParameter('halloween', 'lot of fun');
+     *
+     * @param ContainerBuilder $c
+     * @param LoaderInterface  $loader
+     */
+    abstract protected function configureContainer(ContainerBuilder $c, LoaderInterface $loader);
+
+    /**
+     * {@inheritdoc}
+     */
+    public function registerContainerConfiguration(LoaderInterface $loader)
+    {
+        $that = $this;
```

Comment:
Oh sorry, bly me I did not realized that the problem came from the `configureContainer` visibility :confused: